Benchmark suite is broken

### Version information

`go version go1.10 linux/amd64`

go-git: 43fe6601682d13c5b05a1fc0850e5f294e636486

go-git-fixtures: [a29d269](https://github.com/src-d/go-git/tree/a29d269c3be65e4d1b20c29133c74e0551e1aa5d)

### What did you do?

* clone go-git
* `go get ./...`
* `go test -run=NONE -bench=. ./storage/filesystem/...` 

### What did you expect to see?

Benchmarks pass

### What did you see instead?

```
PASS
ok      gopkg.in/src-d/go-git.v4/storage/filesystem     0.011s
panic: open data/git-7a725350b88b05ca03541b59dd0649fda7f521f2.tgz: no such file or directory

goroutine 34 [running]:
gopkg.in/src-d/go-git-fixtures%2ev3.(*Fixture).DotGit(0x77c640, 0x7, 0x8)
        /go/src/gopkg.in/src-d/go-git-fixtures.v3/fixtures.go:216 +0x4ac
gopkg.in/src-d/go-git.v4/storage/filesystem/internal/dotgit.BenchmarkRefMultipleTimes(0xc42009ed80)
        /go/src/gopkg.in/src-d/go-git.v4/storage/filesystem/internal/dotgit/dotgit_test.go:154 +0x62
testing.(*B).runN(0xc42009ed80, 0x1)
        /snap/go/1473/src/testing/benchmark.go:141 +0xb2
testing.(*B).run1.func1(0xc42009ed80)
        /snap/go/1473/src/testing/benchmark.go:214 +0x5a
created by testing.(*B).run1
        /snap/go/1473/src/testing/benchmark.go:207 +0x80
exit status 2
FAIL    gopkg.in/src-d/go-git.v4/storage/filesystem/internal/dotgit     0.016s
```

--- a/storage/filesystem/internal/dotgit/dotgit_test.go
+++ b/storage/filesystem/internal/dotgit/dotgit_test.go
@@ -151,6 +151,7 @@ func (s *SuiteDotGit) TestRefsFromReferenceFile(c *C) {
 }
 
 func BenchmarkRefMultipleTimes(b *testing.B) {
+	fixtures.Init()
 	fs := fixtures.Basic().ByTag(".git").One().DotGit()
 	refname := plumbing.ReferenceName("refs/remotes/origin/branch")
 

